app/controllers/movie_party_controller.rb

```diff
@@ -12,14 +12,12 @@ def create
     minute = params[:duration][-10..-9].to_i
     movie_party = MovieParty.new(user_id: current_user[:id], movie_title: movie_params[:movie], duration: hour + minute, date: params[:date].values.join('/'), start_time: params[:start_time].values.join(':'))
     friends_ids = params[:friend].keys.map(&:to_i)
-    if movie_party.save
-      friends_ids.each do |number|
-        movie_party.party_users.create!({ movie_party_id: movie_party.id, user_id: number })
-      end
-      redirect_to('/dashboard')
-    else
-      render :new
+    return unless movie_party.save
+
+    friends_ids.each do |number|
+      movie_party.party_users.create!({ movie_party_id: movie_party.id, user_id: number })
     end
+    redirect_to('/dashboard')
```

Comment:
I agree that there isn't a way the movie party won't save, but we'll certainly find out if so